exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/CompositeColumn.kt

```diff
@@ -39,8 +39,12 @@ abstract class BiCompositeColumn<C1, C2, T>(
     /** Transformation that receives the column's composite value and returns the parsed values of the underlying columns. */
     val transformFromValue: (T) -> Pair<C1?, C2?>,
     /** Transformation that receives the retrieved values of [column1] and [column2] and returns a composite value. */
-    val transformToValue: (Any?, Any?) -> T
+    val transformToValue: (Any?, Any?) -> T,
+    nullable: Boolean = false
 ) : CompositeColumn<T>() {
+    init {
+        this.nullable = nullable
+    }
```

Comment:
The alternative could be to make the `nullable` property `protected` in `CompositeColumn` above and perform this set in `CompositeMoneyColumn` directly. I thought it might be good for users that extend this class to have the option to alter nullability.

Reply:
I would wait for request to do this